index.html

```diff
@@ -4,17 +4,176 @@
+                        <img src="images/forum.png" class="h-10 md:pt-2 pt-0" alt="">
+                        <h3 class="text-orange-600 text-2xl font-black ">Forum</h3>
+                        <p class="text-white text-sm ">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ipsum consectetur, ipsam nihil </p>
+                    </div>
+                </li>
+                <li>
+                    <div class="flex items-center bg-slate-300 bg-opacity-10 mx-5 md:mx-0 gap-3 p-2 md:flex-col md:items-center md:text-center md:w-52 md:h-56 md:hover:border-2 md:hover:border-white">
+                        <img src="images/cpu.png" class="h-10 md:pt-2 pt-0" alt="">
+                        <h3 class="text-orange-600 text-2xl font-black ">Tech</h3>
+                        <p class="text-white text-sm ">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ipsum consectetur, ipsam nihil </p>
+                    </div>
+                </li>
+                <li>
+                    <div class="flex items-center bg-slate-300 bg-opacity-10 mx-5 md:mx-0 gap-3 p-2 md:flex-col md:items-center md:text-center md:w-52 md:h-56 md:hover:border-2 md:hover:border-white">
+                        <img src="images/meeting.png" class="h-10 md:pt-2 pt-0" alt="">
+                        <h3 class="text-orange-600 text-2xl font-black ">Meetings</h3>
+                        <p class="text-white text-sm ">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ipsum consectetur, ipsam nihil </p>
+                    </div>
+                </li>
+                <li>
+                    <div class="flex items-center bg-slate-300 bg-opacity-10 mx-5 md:mx-0 gap-3 p-2 md:flex-col md:items-center md:text-center md:w-52 md:h-56 md:hover:border-2 md:hover:border-white">
+                        <img src="images/party.png" class="h-10 md:pt-2 pt-0" alt="">
+                        <h3 class="text-orange-600 text-2xl font-black ">Party</h3>
+                        <p class="text-white text-sm ">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ipsum consectetur, ipsam nihil </p>
+                    </div>
+                </li>
+            </ul>
+            <div class="my-10">
+                <button type="button" class="bg-red-500 text-white text-lg p-5 md:hidden">Join Cardano Student Seminar 2022</button>
+                <a href="#" class="hidden text-white md:block underline">SEE THE WHOLE PROGRAM</a>
+            </div>
+        </div>
+        </section>
+
+        <section class="flex flex-col  bg-brandgray bg-opacity-10">
+        <div class="flex flex-col items-center">
+        <h2 class="text-2xl font-black md:pt-12">Featured Speakers</h2>
+        <div class="flex justify-center mt-2">
+            <div class="bg-brandorange h-px w-10"></div>
+        </div>
+        </div>
+    <div id="speaker" class="">
+       
+    </div>
```

Comment:
- [ ] You did a great work in this section, one thing left is in your design they appear in one column but they have to be in two columns as the original design. They have to appear in one column just on the mobile version.

Your design.
![image](https://user-images.githubusercontent.com/35535130/168817945-f1a4f0c9-57c8-43cf-914a-c76bf2941417.png)

original design
![image](https://user-images.githubusercontent.com/35535130/168818144-7f320120-905b-4384-8b25-cb53e0cc1afa.png)